benefits/static/css/styles.css

```diff
@@ -170,6 +189,9 @@ footer.global-footer .footer-links a:active {
   padding-top: 1px;
   vertical-align: baseline;
   color: transparent;
+  font-size: 16px;
+  font-weight: var(--bold-font-weight);
+  line-height: 1;
```

Comment:
Should `font-size` and `line-height` be variables here?

Reply:
I'm thinking no (at least for now in this PR) b/c this is a one-off 16px link. Once I have a 2nd link that is also 16px, we can make a small-link variable or something.